pkg/parse/parse.go

```diff
@@ -218,14 +494,19 @@ func ValidateVolumeHostDir(hostDir string) error {
 	return nil
 }
 
+// ValidateVolumeCtrDir validates a volume mount's destination directory.
 func ValidateVolumeCtrDir(ctrDir string) error {
+	if len(ctrDir) == 0 {
+		return errors.Errorf("container directory cannot be empty")
+	}
 	if !filepath.IsAbs(ctrDir) {
-		return errors.Errorf("invalid container path, must be an absolute path %q", ctrDir)
+		return errors.Errorf("invalid container path %q, must be an absolute path", ctrDir)
 	}
 	return nil
 }
 
-func ValidateVolumeOpts(option string) error {
+// validates options in --volume flag
+func validateVolumeMountOpts(option string) error {
```

Comment:
Why are you making these private? Doesn't Podman use these?

Reply:
there is another public function `ValidateVolumeOpts` for podman. Make `validateVolumeMountOpts` private for buildah volume. Because podman volume doesn't have `:o` option but buildah has

Reply:
I plan on adding the :o option for Podman, so I want this public as well.

Reply:
sure